Feat: http retrieval as experimental feature (#10772)
ipfs/kubo: docs/config.md

Type: `array[peerID]`

### `Routing.DelegatedRouters`

> This is an array of URL hostnames that support the [Delegated Routing V1 HTTP API](https://specs.ipfs.tech/routing/http-routing-v1/) will be used alongside the DHT when [`Routing.Type`](#routingtype) is set to `auto` or `autoclient`.

Reviewer: ```suggestion
This is an array of URL hostnames that support the [Delegated Routing V1 HTTP API](https://specs.ipfs.tech/routing/http-routing-v1/) which are used alongside the DHT when [`Routing.Type`](#routingtype) is set to `auto` or `autoclient`.
```
